api/api.go

```diff
@@ -77,6 +78,31 @@ func (response GQLResponseErrors) ToError() error {
 	return errors.New(strings.Join(messages, ": "))
 }
 
+// GraphQLServerAddress returns the full address to CircleCI GraphQL API server
+func GraphQLServerAddress() (string, error) {
```

Comment:
I would make this function take the `host` and `endpoint` as arguments, so that it is a pure function, and write a second help-function that plucks the value from viper. This will mean that you can test one function without the other.